src/gateway_api/endpoints/transaction_endpoints.rs

```diff
@@ -0,0 +1,47 @@
+use crate::prelude::*;
+
+impl GatewayClient {
+    /// Get Construction Metadata
+    ///
+    /// Returns information needed to construct a new transaction including current epoch number.
+    ///
+    /// See [the Gateway API docs for details][doc].
+    ///
+    /// [doc]: https://radix-babylon-gateway-api.redoc.ly/#operation/TransactionConstruction
+    pub(crate) async fn transaction_construction(&self) -> Result<LedgerState> {
+        self.post_empty(
+            "transaction/construction",
+            |response: TransactionConstructionResponse| {
```

Comment:
and in `src/gateway_api/endpoints/transaction_endpoints` all `transaction/` endpoints go...